### DISCUSSION TITLE
What import alias would you like confiugred?

### DISCUSSION THREAD

**Discussion by patilharsh03** [vercel/next.js#46087] [Help] [answered]
When I am creating the next app these options are coming to me. I never see this in any tutorial or any person who uses Next.js. can someone explain to me what it is? Is this a new thing on Next.js 13?? or I am only seeing these ??

![image](https://user-images.githubusercontent.com/58598919/219853509-7073ad5a-72ca-4f65-a2f4-305e8fb76f8a.png)

**Comment by frencojobs** [comment, marked as answer]
It's asking if you'd like to configure TypeScript [path mappings](https://www.typescriptlang.org/tsconfig#paths). If you chose to use TypeScript, then it'll create the mappings in `tsconfig.json`, otherwise it'll create a `jsconfig.json` and put the mappings there. They're not a new thing of Next.js 13, but that configuration option is fairly recent in create-next-app which is probably why you're only seeing this now.
